README.md

```diff
@@ -119,6 +119,49 @@ Connection: keep-alive
 
 This error shows the request is missing a required property `name` from the HTTP request body.
 
+### HTTP API
+
+Prism provides [HTTP API][http-docs] as separate module for forwarding and mocking requests.
+
+#### Forwarding example
+
+```ts
+const data = await forwarder.forward({
```

Comment:
FYI @mallachari we should not be using this module as a standalone tool anyway. I will describe how to use it in a bit (not that if used this way you won't get stuff like input/output validation)